packages/ui-dom-utils/src/getComputedStyle.ts

```diff
@@ -40,16 +40,21 @@ import type { UIElement } from '@instructure/shared-types'
  * @param { string | null | undefined } pseudoElt - A string specifying the pseudo-element to match. Omitted (or null ) for real elements.
  * @returns { Object } object containing css properties and values for the element
  */
-function getComputedStyle(el?: UIElement, pseudoElt?: string | null) {
-  let style = {}
+function getComputedStyle(
+  el?: UIElement,
+  pseudoElt?: string | null
+): CSSStyleDeclaration {
+  let style = {} // TODO(breaking) return undefined instead!
```

Comment:
this is a bug, but we can only fix it with a major release. sometimes just an empty object is returned, in this case functions that call this one can fail, e.g. in `findFocusable`: `getComputedStyle(element).getPropertyValue('position')`